vmm/src/acpi.rs

```diff
@@ -378,6 +378,8 @@ pub fn create_acpi_tables(
     guest_mem
         .write_slice(dsdt.as_slice(), dsdt_offset)
         .expect("Error writing DSDT table");
+    #[cfg(target_arch = "aarch64")]
+    tables.push(dsdt_offset.0);
```

Comment:
Why is this aarch64 specific? Does the DSDT actually need to be in the XSDT?

Reply:
Thanks for reminding this.
This is a early test code. We need a fix on EDK2 code.
I will change it back when EDK2 is ready. Before that I will change the PR draft, in case it's merged unexpectedly.

Reply:
This is removed now. Thanks for @jongwu fixing the issue in EDK2.

The PR is ready for review again.